python-client/pyproject.toml

```diff
@@ -18,32 +24,27 @@ url = "https://download.pytorch.org/whl/cpu/torch_stable.html"
 name = "torch"
 
 
-
 [tool.pdm.scripts]
 # add "-n auto" to the pytest command to parallelize the execution
 test.cmd = "pytest -c pyproject.toml tests --cov=giskard --cov-report=xml --disable-warnings --no-header -vv --durations=0"
 test.env = { GSK_DISABLE_ANALYTICS = "True" }
 # for some reason github runners don't work when calling 'pdm test -m "not slow"'
+
 test-fast.cmd = "pytest -n auto -m 'not slow' -c pyproject.toml tests --cov=giskard --cov-report=xml --disable-warnings --no-header -vv --durations=0"
 test-fast.env = { GSK_DISABLE_ANALYTICS = "True" }
 lint = "ruff giskard tests"
-doc = "python -m sphinx_autobuild --watch giskard docs docs/_build/html"
-
+doc = "sphinx-build docs docs/_build/html"
+watch-doc = "python -m sphinx_autobuild --watch giskard docs docs/_build/html"
+clean = "rm -rf .venv coverage.xml coverage*"
+check-deps = "deptry ."
 [tool.pdm.dev-dependencies]
 dev = [
     "typing-extensions>=4.5.0",
-    "bandit>=1.7.4",
-    "darglint>=1.8.1",
     "jupyter>=1.0.0",
     "jupyterlab>=3.4.2",
     "pre-commit>=2.19.0",
-    "pydocstyle>=6.1.1",
-    "pylint>=2.13.9",
-    "pyupgrade>=2.32.1",
-    "safety>=1.10.3",
     "mypy>=0.982",
     "deptry>=0.5.13",
-    "httpretty>=1.1.4",
```

Comment:
was it `deptry` who sugested the removal?

Reply:
No, this one I checked it manually I think.

I used deptry, but I also had a look around on deps to see if some were unused.